aqueductcore/frontend/src/helper/auth/userScope.tsx

```diff
@@ -39,3 +39,24 @@ export function isUserAbleToDeleteExperiment(userInfo: UserInfo, createdBy: Expe
     return false
 }
 
+export function isUserAbleToCreateTask(userInfo: UserInfo) {
+    for (const item of userInfo.scopes) {
+        if (item === defined_scopes.JOB_CREATE) {
+            return true
+        }
+    }
+    return false
+}
```

Comment:
I guess this can be simplified to

```suggestion
export function isUserAbleToCreateTask(userInfo: UserInfo) {
    return userInfo.scopes.some((scope) => scope === defined_scopes.JOB_CREATE);
}
```